fixmydjango/fixmydjango
# Refs #47 - Stretch the comment box to fill external box

Link to task: https://github.com/fixmydjango/fixmydjango/issues/47
How to test: 
- Enter in the page to submit an error or in the page of an error 
- You should the see the comment box stretched

In the Error page:
![error_page_comment_box](https://cloud.githubusercontent.com/assets/8882260/23260919/e2b29734-f9b2-11e6-9bf6-97e68e51a3b8.png)

In the submit error page:
![submit_error_page](https://cloud.githubusercontent.com/assets/8882260/23260951/05d32fa8-f9b3-11e6-99a9-72d74b7fbf3d.png)

When the pr https://github.com/fixmydjango/fixmydjango/pull/96 get merged, the preview box is going to stretch as well:

![preview_box](https://cloud.githubusercontent.com/assets/8882260/23262588/0c48bfcc-f9ba-11e6-86cb-10e969d6555c.png)
